libbeat/docs/outputconfig.asciidoc

```diff
@@ -198,7 +198,7 @@ operation doesn't succeed after
 ===== bulk_max_size
 
 The maximum number of events to bulk in a single Elasticsearch bulk API index request.
-The default is 50.
+The default is 200.
```

Comment:
I've created a new doc issue #586 to track the doc changes for bulk_max_size so these comments don't get lost under the outdated diff....